projects/hslayers/src/components/add-data/file/file-base.component.ts

```diff
@@ -53,28 +55,28 @@ export class HsAddDataFileBaseComponent
   }
 
   ngOnInit(): void {
-    const commonFileServiceRef = this.hsAddDataCommonFileService.get(this.app);
-    commonFileServiceRef.dataObjectChanged
+    this.app = this.hsConfig.id;
```

Comment:
I think there are some document queries in the process so to distinguish between instances of uploader `[uploader]="'hs-file-raster' + app"` when this would be opened in more apps at once. Same applies for layer checkboxes

Reply:
Ah, I can see, you are right!